graphql-server/src/remotes/remote.model.ts

```diff
@@ -45,3 +66,18 @@ export function getRemoteListRes(
     nextOffset: getNextOffset(remotes.length, args.offset ?? 0),
   };
 }
+
+export function fromPullRes(r: RawRow): PullRes {
+  return {
+    fastForward: r.fast_forward !== "0",
```

Comment:
I think this should both be `r.fast_forward === "1"`. In case some weird int gets in there you want it to be false in all circumstances except for 1